lib/process-services-cloud/src/lib/services/notification-cloud.service.spec.ts

```diff
@@ -43,39 +38,25 @@ describe('NotificationCloudService', () => {
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [ProcessServiceCloudTestingModule]
+            imports: [ProcessServiceCloudTestingModule],
```

Comment:
CAn we get rid of this ProcessServiceCloudTestingModule? this is importing the whole Core